source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ColumnRenderer.java

```diff
@@ -42,6 +42,12 @@ public static ColumnRenderer getInstance()
         return s_instance;
     }
 
+    public void setInformationForColumnRendering(boolean isFirstColumn, boolean isLastColumn)
```

Comment:
>setInformationForColumnRendering [](start = 16, length = 32)

I think I'd prefer two methods here -- `setIsFirstColumn()` and `setIsLastColumn()`